Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPrintDialog.cs

```diff
@@ -79,7 +79,7 @@ public bool ShowIcon
             _commonDialogHandler = new CommonDialogHandler(true)
             {
                 ClickCallback = ClickCallback,
-                Icon = Resources.CommonDialogIcons.printer,
+                Icon = CommonDialogIcons.printer_green,
```

Comment:
Did not use green, due to colour blind requirements (Against the blue and red!)

Reply:
I can change it to the orange version, I will also change the font icon, as that's green as well